reference/carts.v3.yml

```diff
@@ -3847,7 +3883,7 @@ components:
             - description
             - owner_client_id
             - date_created
-            - date_modified
+            - date modified
```

Comment:
@bc-traciporter  I haven't tested it myself, but it seems odd that this enum value isn't `date_modified` (the other enum values have underscores instead of spaces)

Reply:
@bc-andreadao you are correct. This should be date_modified. I had to change it in the description text. I made a mistake with this one. Thanks for finding it. I will fix it.